packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -305,7 +305,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
     _updateMoveAnimation();
   }
 
-  AnimationController? _moveController;
+  late final AnimationController _moveController;
```

Comment:
`_moveController` is initialized in `initState` and isn't reassigned afterward, so it can be non-nullable.

Reply:
`late final` is inherently less safe than a nullable type, as the compiler won't force you to check if it's non-null before you de-reference it.

Reply:
Thanks, that's good to know. Just pushed [`b3af738`](https://github.com/flutter/flutter/pull/150276/commits/b3af7383ff44b4d827b422bc4469f8d52fa58efe) to mitigate the issue.

Reply:
:+1: